sdkkinds/dashboard/dashboard_v2.cue

```diff
@@ -0,0 +1,794 @@
+package dashboard
+
+v2: {
+	"v2alpha1": {
+		schema: {
+			spec: #DashboardSpec
+		}
+	}
+}
+
+#DashboardSpec: {
```

Comment:
@dprokop FYI, this is a snapshot of the V2 schema (https://github.com/grafana/grafana/blob/main/packages/grafana-schema/src/schema/dashboard/v2alpha0/dashboard.schema.cue) from a few weeks ago.